aries-site/src/pages/components/button.mdx

```diff
@@ -185,3 +185,54 @@ Button label text size may be specified. Padding and rounding scale proportionat
 >
   <ButtonSizingExample />
 </Example>
+
+## Responsiveness
+
+Ensure you are always being mindful of our responsive page layouts when designing user experiences. Even if you don’t anticipate your users leveraging mobile or tablet devices, they may be reducing/expanding browser widths, zooming in on content, or manipulating the browser in unanticipated ways. As a best practice, always design for a usable UI on all browser widths and resolutions.
+
+### Best Practices
+- Ensure the button label does not wrap, truncate, or alter the text content in any way when presented on smaller browser widths.
+- Button label text should not scale down to fit on smaller browser widths.
+- The minimum button width on mobile should be 42px.
+
+### Call to Action
+
+When buttons are “Brand” or “CTA (Call to Action)”, it is best to have them fill the full width of the container for browser widths of 576px (S) or smaller. Because press target size and an unclear visual hierarchy are key concerns here, this utilizes available space to ensure users can easily identify and interact with important buttons.
```

Comment:
The "full width of the container for browser widths of 576px (S) or smaller" kind of gets lost. I think the reader's eye will go straight to the examples and apply the full-width no matter what. --> Need to re-think how this concept is presented.

Prefer references to design tokens, in this case "small" rather than pixels. 

```suggestion
When buttons are “Brand” or “Call to Action (CTA)”, it is best to have them fill the full width of the container for browser widths of "small". Because press target size and an unclear visual hierarchy are key concerns here, this utilizes available space to ensure users can easily identify and interact with important buttons.
```